internal-services/catalog/iib-pipeline.yaml

```diff
@@ -19,11 +19,9 @@ spec:
     - name: iibServiceConfigSecret
       type: string
       description: Secret containing IIB service Config
-      default: iib-services-config
```

Comment:
removing the `default` is desired in this PR?

Reply:
nevermind. I see the reason in the other PR

Reply:
Not really. I removed to test and forgot to put them back.